.gitignore

```diff
@@ -48,9 +48,6 @@ vendor/bundle/
 # produced by tests (?)
 .rakeTasks
 
-# produced by tests (?)
-*.csv
```

Comment:
Needed to be able to move csv file fixtures. Anything produced by specs should go into `tmp/`, anyway.

Reply:
legit